app.rb

```diff
@@ -180,6 +180,36 @@ class << self
     CategoryContribution.all.to_json
   end
 
+  get '/api/payments/?:id?' do |id|
+    # TODO: Figure out how to cache this
+    # cache_control :public
+    # last_modified Import.last.import_time
```

Comment:
You can uncomment these, since for a given URL, the results should not change between imports.